### DISCUSSION TITLE
Should we update our repo devcontainer's image?

### DISCUSSION THREAD

**Discussion by edchanonline** [earthlab-education/Earth-Analytics-2023-01-Intro#421] [Q&A04 Raster Data] [answered]
Having some issues in importing `earthpy` in my portfolio repo. Should we use a different image that's defined in our devcontainer file? Thank you!

Current image in use:
`registry.hub.docker.com/earthlab/earth-analytics-python-env:latest`

Image from this week's lesson:
`registry.hub.docker.com/earthlab/earth-analytics-dev-env:latest`

**Comment by eculler** [comment, marked as answer]
@eggvoice and @earthlab-education/ea-students yes, if you are using the AppeearsDownloader you will need to update your portfolio's .devcontainer.json to use the earth-analytics-dev-env environment, as it has not yet been released.

**Reply by edchanonline** [reply]
Thank you!!

**Reply by lijo8146** [reply]
Will someone please explain the answer/solution in the context of someone who is not familiar with Docker.

**Reply by lucap1211** [reply]
I would also appreciate clarification on this issue. I tried to edit my devcontainer.json in my repository and changed the current image to the lesson's image that uses earthpy ( registry.hub.docker.com/earthlab/earth-analytics-dev-env:latest ) for the assignment.  I still get an error message: "ModuleNotFoundError: No module named 'earthpy.appeears'". 

Also; I was wondering if I should change the image back to the original one after I finish using earthpy?

**Comment by edchanonline** [comment]
@lucap1211 @lijo8146 

I haven't implemented this but this should work in theory. 
1. Copy the latest devcontainer e.g. https://github.com/earthlab-education/bootcamp-04-vector-eggvoice/blob/main/.devcontainer.json (use your usename instead of `eggvoice`)
2. Replace the one in your portfolio repo
3. Create a new Codespace (so the new Codespace is built with the new devcontainer requirements)
